package.json

```diff
@@ -3,14 +3,20 @@
   "version": "0.0.0",
   "private": "true",
   "scripts": {
-    "lint": "npx eslint packages/*/src/**/*.ts --fix",
-    "clean-packages": "npx lerna run clean-packages && rm -rf node_modules",
-    "build": "npx lerna run build --stream --include-dependencies",
+    "lint": "eslint packages/*/src/**/*.ts --fix",
+    "clean-packages": "lerna run clean-packages && rm -rf node_modules",
+    "build": "lerna run build --stream --include-dependencies",
     "build-and-test": "npm run build && npm run test",
     "dev": "lerna watch --include-dependencies -- npm run build",
     "dev-test": "lerna watch --include-dependencies -- npm run build-and-test",
-    "coverage": "npx lerna run --include-dependencies --stream coverage",
-    "test": "npx lerna run --include-dependencies --stream test",
+    "coverage": "run-s coverage:indexdb coverage:inmemory coverage:leveldb",
+    "coverage:indexdb": "npm --prefix packages/indexdb run coverage",
+    "coverage:inmemory": "npm --prefix packages/inmemory run coverage",
+    "coverage:leveldb": "npm --prefix packages/leveldb run coverage",
+    "test": "run-s test:indexdb test:inmemory test:leveldb",
+    "test:indexdb": "npm --prefix packages/indexdb run test",
+    "test:inmemory": "npm --prefix packages/inmemory run test",
+    "test:leveldb": "npm --prefix packages/leveldb run test",
```

Comment:
I got zero time to waste on `lerna` swallowing errors.
I feel sad about this, but I also have a life to live